sharding/indexHashedNodesCoordinatorWithRater.go

```diff
@@ -55,6 +55,22 @@ func (ihgs *indexHashedNodesCoordinatorWithRater) SetNodesPerShards(
 	return ihgs.expandAllLists(epoch)
 }
 
+// ComputeLeaving - computes the validators that have a threshold below the minimum rating
+func (ihgs *indexHashedNodesCoordinatorWithRater) ComputeLeaving(allValidators []Validator) []Validator {
+	leavingValidators := make([]Validator, 0)
+	for _, validator := range allValidators {
```

Comment:
`validator` name for this block variable shadows another, can you please rename it?

Reply:
fixed